pages/[id]/edit.js

```diff
@@ -0,0 +1,51 @@
+import { StyleList } from "@/components/StyledList.js";
+import FormActivityForEdit from "@/components/FormEdit.js";
```

Comment:
it can be very confusing if you rename default exports ;)
```suggestion
import FormEdit from "@/components/FormEdit.js";
```